app/Http/Requests/AdminNewUpdateTraderRequest.php

```diff
@@ -29,6 +29,8 @@ public function rules()
             'name' => 'required|string|max:160',
             // MUST be present, integer, in table
             'market' => 'required|integer|exists:markets,id',
+            // Optional, can be null, can be [yes,on,1,true]
+            'disabled' => 'nullable|in:yes,on,1,true',
```

Comment:
if I use "accepted" it MUST be present, which is not the usual expectation for an optional checkbox.